WebKitBrowser/Apps.conf.in

```diff
@@ -1,5 +1,6 @@
 precondition = ["Internet"]
 autostart = "@PLUGIN_APPS_AUTOSTART@"
+startuporder = "@PLUGIN_APPS_STARTUPORDER@"
```

Comment:
Should this not be conditional? Only set it if PLUGIN_APPS_STARTUPORDER is set. In the "old" configuration generation it is conditional.

Reply:
In new config generator case, if a flag is empty then it will not generate empty config for that particular flag. So conditional check is not required

Reply:
ahh oke..